{
  "comments": [
    {
      "key": {
        "uuid": "4d0ab91e_e9226e64",
        "filename": "liblog/logd_reader.cpp",
        "patchSetId": 3
      },
      "lineNbr": 428,
      "author": {
        "id": 1003224
      },
      "writtenOn": "2019-10-08T19:17:46Z",
      "side": 1,
      "message": "you\u0027re not assigning `errno \u003d 0` anywhere, so this seems invalid?\n\n(which makes me realize the same issue [indirectly] with L396.)",
      "range": {
        "startLine": 428,
        "startChar": 19,
        "endLine": 428,
        "endChar": 29
      },
      "revId": "27bb63b905e1ec5a7a618da8ef15c1c655a7900a",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "ffb02823_7449d448",
        "filename": "liblog/logd_reader.cpp",
        "patchSetId": 3
      },
      "lineNbr": 428,
      "author": {
        "id": 1064128
      },
      "writtenOn": "2019-10-08T19:24:38Z",
      "side": 1,
      "message": "I\u0027ll add that.  Is it strictly required though?  If recv() returns -1; it should always set errno, no?",
      "parentUuid": "4d0ab91e_e9226e64",
      "range": {
        "startLine": 428,
        "startChar": 19,
        "endLine": 428,
        "endChar": 29
      },
      "revId": "27bb63b905e1ec5a7a618da8ef15c1c655a7900a",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "5d57f7c2_84aea53e",
        "filename": "liblog/logd_reader.cpp",
        "patchSetId": 3
      },
      "lineNbr": 428,
      "author": {
        "id": 1064128
      },
      "writtenOn": "2019-10-08T19:30:28Z",
      "side": 1,
      "message": "Unless you mean that checking that errno !\u003d 0 is useless, which makes a lot more sense, especially given my own comment that if recv returns -1, errno should always be non-zero.",
      "parentUuid": "ffb02823_7449d448",
      "range": {
        "startLine": 428,
        "startChar": 19,
        "endLine": 428,
        "endChar": 29
      },
      "revId": "27bb63b905e1ec5a7a618da8ef15c1c655a7900a",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "a9e32572_44e595db",
        "filename": "liblog/logd_reader.cpp",
        "patchSetId": 3
      },
      "lineNbr": 428,
      "author": {
        "id": 1003224
      },
      "writtenOn": "2019-10-08T19:35:31Z",
      "side": 1,
      "message": "yeah, whatever you\u0027re trying to say, \"this isn\u0027t it\" :-)\n\ni think you just want to delete the errno check in both cases. it might be helpful to rename `saved_errno` to `write_errno` in the L396 case, since \"saved_errno\" sounds like the ErrnoRestorer idiom, which this _isn\u0027t_.",
      "parentUuid": "5d57f7c2_84aea53e",
      "range": {
        "startLine": 428,
        "startChar": 19,
        "endLine": 428,
        "endChar": 29
      },
      "revId": "27bb63b905e1ec5a7a618da8ef15c1c655a7900a",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "602be576_5d8d8372",
        "filename": "liblog/logd_reader.cpp",
        "patchSetId": 3
      },
      "lineNbr": 428,
      "author": {
        "id": 1064128
      },
      "writtenOn": "2019-10-08T20:11:08Z",
      "side": 1,
      "message": "Done",
      "parentUuid": "a9e32572_44e595db",
      "range": {
        "startLine": 428,
        "startChar": 19,
        "endLine": 428,
        "endChar": 29
      },
      "revId": "27bb63b905e1ec5a7a618da8ef15c1c655a7900a",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": false
    }
  ]
}